src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -71,7 +71,7 @@ private partial class SymbolInlineRenameInfo : IInlineRenameInfoWithFileRename
                 _fallbackOptions = fallbackOptions;
                 this.RenameSymbol = renameSymbol;
 
-                this.HasOverloads = RenameLocations.GetOverloadedSymbols(this.RenameSymbol).Any();
+                this.HasOverloads = RenameUtilities.GetOverloadedSymbols(this.RenameSymbol).Any();
```

Comment:
we had static helper methods scattered around 3 places.  If the static method was called in at least 2 distinct locations, i moved to RenameUtilities since htat already exists and seems more conducive to that purpose.